lib/helper/Nightmare.js

```diff
@@ -759,7 +759,7 @@ class Nightmare extends Helper {
       context = this.context;
     }
     let locator = guessLocator(context) || { css: context};
-    this.browser.optionWaitTimeout = sec * 1000 || this.options.waitForTimeout;
+    this.browser.options.waitTimeout = sec * 1000 || this.options.waitForTimeout;
```

Comment:
yeah, i see the bug. It was due different implementations in WebDriverIO, SeleniumWebDriver and Nightmare